pages/common/git-squash.md

```diff
@@ -0,0 +1,21 @@
+# git squash
+
+> Squash N last changes up to a ref'ed commit into current HEAD.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-squash>.
+
+- Squash everything from a specific branch into the current branch:
+
+`git squash {{source-branch}}`
+
+- Squash everything from a commit reference into the current branch:
```

Comment:
The original sounds like it would squash all commits from all branches.
```suggestion
- Squash all commits starting with a specific commit on the current branch:
```
Also, this replaced all those commits with the squashed one right?

Reply:
That's right, at least in my observations testing the command to check the behaviour.